content/it/partners/_index.html

```diff
@@ -1,135 +1,43 @@
 ---
-draft: True
-title: Partners
-bigheader: Kubernetes Partners
-abstract: Growing the Kubernetes ecosystem.
+title: Partner
+bigheader: I partner di Kubernetes
+abstract: Far crescere l'ecosistema di Kubernetes.
 class: gridPage
 cid: partners
 ---
 
 <section id="users">
-    <main>
-        <h5>Kubernetes collabora con i partner per creare una base di codice forte e vivace che supporti uno spettro di piattaforme complementari.</h5>
-        <div class="col-container">
-          <div class="col-nav">
-            <center>
-              <h5---
-title: Partners
-bigheader: I Partners di Kubernetes
-abstract: La Crescita dell'ecosistema Kubernetes.
-class: gridPage
-cid: partners
----
-<meta charset="utf-8" />
-
-<section id="users">
-    <main>
-        <h5>Kubernetes lavora in collaborazione con diversi partner al fine di creare un codice dalla base forte e dinamica che supporti uno spettro di piattaforme complementari.</h5>
+    <main class="main-section">
+        <h5>Kubernetes collabora con i partner per creare un sorgente forte e vivace che supporti uno spettro di piattaforme complementari.</h5>
         <div class="col-container">
           <div class="col-nav">
             <center>
               <h5>
-                <b>Fornitori di Servizi Certificati Kubernetes</b>
+                <b>Fornitori Certificati di Servizi su Kubernetes</b>
```

Comment:
What about: "Fornitori di servizi Certificati su Kubernetes"

Reply:
are you sure about this? I think that "Certified Service Providers" means that the provider is certified, not the service (that usually is consulting, so very difficult to certify)

Reply:
"di servizi" in the middle between Certified and Kubernetes feels weird to me.
But we ca iterate later in the final review